src/components/ModalViewContainer/index.tsx

```diff
@@ -12,13 +12,8 @@ import CancelAlert from '../forms/CancelAlert';
 declare const PCore;
 
 // Utility to determine if a JSON object is empty
-function isEmptyObject(inObj: Object): Boolean {
-  let key: String;
-  // eslint-disable-next-line guard-for-in
-  for (key in inObj) {
-    return false;
-  }
-  return true;
+function isEmptyObject(obj: Object): Boolean {
+  return Object.keys(obj).length === 0;
```

Comment:
Should we put isEmptyObject() in some utility file instead?

Reply:
as of now, this method was used in this component itself so placed here itself

Reply:
That's alright but this seems like a generic method which can be utilised in other components as well in future. But we can do that later, no issues.